lahja/base.py

```diff
@@ -221,12 +221,15 @@ def name(self) -> str:
     async def wait_stopped(self) -> None:
         await self._stopped.wait()
 
+    @property
     def is_running(self) -> bool:
-        return not self.is_stopped and self.running.is_set()
+        return not self.is_stopped and self._running.is_set()
```

Comment:
I'm really confused by this. How is it possible that neither `mypy` nor `flake8` caught this?

Reply:
In my local env. just pushed something to see if CI catches it.

Reply:
it did https://circleci.com/gh/pipermerriam/lahja/2245

Reply:
Yes, it seems it catches it *after* this change but not on current master which is really weird. Also, the `@property` decorators also need to be applied on the `RemoteEndpointAPI`.

Reply:
In fact, as soon as you remove the `@property` from `is_stopped` it won't catch this error anymore. This feels like a weird mypy bug to me.

Reply:
That's very odd.

Reply:
Ok, here is a minimal test case.

```python
class TestCase:

    def foo(self) -> bool:
        return False and self.meh()
```

This is not caught by `mypy`. Do you think this is expected behavior?

Reply:
I opened an issue against mypy https://github.com/python/mypy/issues/6990